src/js/jsx/sections/libraries/assets/CharacterStyle.jsx

```diff
@@ -84,21 +86,20 @@ define(function (require, exports, module) {
                 fontColorHex = tinycolor(color.value).toHexString().toUpperCase();
             }
 
-            var fontSizeAndColorStr = _.remove([fontSizeStr, fontColorHex], null).join(", ");
-
-            var classNames = classnames("libraries__asset", {
-                "assets__graphic__dragging": this.props.isDragging,
-                "libraries__asset-selected": this.props.selected
-            });
-
-            var displayName = element.displayName !== "" ? element.displayName : font.name + " " + font.style,
+            var fontSizeAndColorStr = _.remove([fontSizeStr, fontColorHex], null).join(", "),
+                displayName = element.displayName !== "" ? element.displayName : font.name + " " + font.style,
```

Comment:
empty string is considered as `false` in JS, so I simplified this line to `element.displayName ||  (font.name + " " + font.style)`